ftw/monitor/command.py

```diff
@@ -27,3 +17,12 @@ def monitor(zope2Cmd, *args):
     else:
         content = ' '.join(args)
     netcat('127.0.0.1', int(monitor_port), '%s\n' % content)
+
+
+def dump_perf_metrics():
+    """Collect and dump performance metrics for all instances in
+    InfluxDB line protocol format.
+    """
+    deployment_dir = get_deployment_dir()
+    metrics = dump_all_perf_metrics(deployment_dir)
```

Comment:
Could be printed directly instead of assigning it to a variable first.

Reply:
👍  fixed, thanks!